### DISCUSSION TITLE
小地图不显示

### DISCUSSION THREAD

**Discussion by zfs0721** [FrozenFish259/Goose_Goose_Duck_Hack#148] [Q&A 疑难解答区] [answered]
小地图不显示

**Comment by FrozenFish259** [comment, marked as answer]
截图

**Reply by zfs0721** [reply]
![HP27V22XMUSV_NKBGOZ7OW1](https://user-images.githubusercontent.com/104275044/213099316-e5969231-c2b3-4739-820d-9d34f799bc4e.png)

**Reply by FrozenFish259** [reply]
你没有把img文件夹放在辅助同目录所以找不到图片

**Reply by zfs0721** [reply]
OOO好嘞 谢谢

**Reply by RollXo** [reply]
放了

**Reply by RollXo** [reply]
找到原因了，是被windows安全中心删了，笑
